doc/2/api/protocols/http/index.md

```diff
@@ -64,3 +64,24 @@ The listening port can be modified under the `server.port` section of the [confi
 HTTP and WebSocket protocols share the same underlying server instance.  
 Modifying the listening port will impact these two protocols.
 :::
+
+<SinceBadge version="auto-version"/>
+### HTTP-only options
+
+Only the HTTP protocol does support the following options in a [KuzzleRequest](/core/2/api/payloads/request)
+
+| Property     | Type               | Description                                                                           |
+| ------------ | ------------------ | ------------------------------------------------------------------------------------- |
+| `cookieAuth` | <pre>boolean</pre> | Instructs Kuzzle to store / use authentication tokens within secured HTTP Cookies     |
```

Comment:
well... now that option is only available for a few actions in the `auth` controller (login, checkToken, refreshToken).

So this piece of documentation should be removed from global HTTP options, and instead be added to each one of those routes.